addons/website_sale/data/data.xml

```diff
@@ -80,13 +81,15 @@ for product in products:
     res_product['list_price'] = FieldMonetary.value_to_html(res_product['price'], monetary_options)
     data = get_rendering_data_structure()
     for field_name in dynamic_filter.field_names.split(","):
+        if field_name in ['list_price']: # already rendered
```

Comment:
in this case just need a t-else type, but still not sure to understand what you are trying to fix here ? 
Basically, product snippet is done to work 'out of the box', so what is the gain with this code ?

Reply:
From what I understand, the widget is written in a dynamic way to enable specifying which data fields must be displayed by looking up the `field_names` field of the selected filter.
If this is the case, used fields could be obtained from a customization, maybe from customer reviews attached to the product (imagine a `product.best_customer_quote`), such a quote could be modified by public users on the website.
Such a field might not be safe to use without escaping, which is the reason I made all non-binary fields pass through the rendering layer. The problem was then that the `list_price` field had already been rendered, so it got wrongly re-escaped.